### PR TITLE
Fix of issue #992

### PR DESCRIPTION
**Pull request by CrosRoad95**
It fix issue #992, now guiCreateStaticImage throw warning if image don't exists

Also this will throw error for default admin panel due path `gui\\images\\dot.png` is wrong, correct is `client\\images\\dot.png` 

image

```lua
guiCreateStaticImage(0.35, 0.68, 0.1, 0.1, "consider.png", true)
guiCreateStaticImage(0.35, 0.78, 0.1, 0.1, "invalidImage.png", true) // throw error
```

Full test resource:
[static image.zip](https://github.com/multitheftauto/mtasa-blue/files/3473065/static.image.zip)

### CODE REVIEW

File: Client/mods/deathmatch/logic/luadefs/CLuaGUIDefs.cpp [hunk 1]
```diff
@@ -795,11 +795,13 @@ int CLuaGUIDefs::GUICreateStaticImage(lua_State* luaVM)
             if (CResourceManager::ParseResourcePathInput(path, pResource, &strPath))
             {
                 CClientGUIElement* pGUIElement = CStaticFunctionDefinitions::GUICreateStaticImage(*pLuaMain, position, size, strPath, relative, parent);
-                lua_pushelement(luaVM, pGUIElement);
-                return 1;
+                if (pGUIElement != NULL)
```
Comment (Citizen01) [inline Client/mods/deathmatch/logic/luadefs/CLuaGUIDefs.cpp]: IIRC, we now try to use `nullptr` instead of `NULL`

Comment (CrosRoad95) [reply Client/mods/deathmatch/logic/luadefs/CLuaGUIDefs.cpp]: i don't know which variant i should use

**Comment by qaisjp** [comment]
Why didn't this work before?

**Comment by CrosRoad95** [comment]
`m_pScriptDebugging->LogCustom(luaVM, argStream.GetFullErrorMessage());`
wasn't called due `return 1`

**Comment by qaisjp** [comment]
But [the first `if`](https://github.com/multitheftauto/mtasa-blue/pull/1041/files#diff-16d0409384a35f36d21666b5ced6b2caL795) shouldn't let it get to that point

**Comment by CrosRoad95** [comment]
it checks if path is valid, not does file exists

**Comment by qaisjp** [comment]
what happens with your pull request if you try to create a staticimage with a file that is empty?

i believe your code currently blames `CStaticFunctionDefinitions::GUICreateStaticImage` returning null for any reason on a bad filepath

you should add an if block to check if the file exists instead of doing a null check. you could even output a generic "failed to create static image" warning when null is returned

**Comment by CrosRoad95** [comment]
now it working as present:
```lua
guiCreateStaticImage(0.35, 0.68, 0.1, 0.1, "c.lua", true) -- Failed to create static image
guiCreateStaticImage(0.35, 0.68, 0.1, 0.1, "consider.png", true) -- no errors, image got created
guiCreateStaticImage(0.35, 0.78, 0.1, 0.1, "doesntExists.png", true) -- File not found
guiCreateStaticImage(0.35, 0.78, 0.1, 0.1, ":asd:asd:asd", true) -- Bad file path
```

File: Client/mods/deathmatch/logic/luadefs/CLuaGUIDefs.cpp (hunk 1 above)
Comment (qaisjp) [reply Client/mods/deathmatch/logic/luadefs/CLuaGUIDefs.cpp]: never use `NULL`, always use `nullptr`

Comment (CrosRoad95) [reply Client/mods/deathmatch/logic/luadefs/CLuaGUIDefs.cpp]: done

**Comment by qaisjp** [comment]
thank you

**Pull request merged**